### PR TITLE
Allowing the 'null' value for implementation report.

### PR DESCRIPTION
**Pull request by iherman**
This is to fulfill the request of @bduga: implementation reports allow for the value of `null`, resulting in a `n/a` cell in the report. This is also the default value in the implementation report template.

The PR also includes some additional minor changes:

- removal of the test report created during the live demonstration
- minor changes on the respec headers to avoid unnecessary respec warnings and inconsistencies in the generated header

### CODE REVIEW

**Comment by iherman** [comment]
@dlazin I would like to merge this PR as soon as possible to avoid unnecessary merge conflicts if new tests come in...

File: docs/drafts/index.html [hunk 1]
```diff
@@ -7,13 +7,13 @@
...
 				specStatus: "base",
 				noRecTrack: true,
-				publishDate: "2022-04-22",
+				publishDate: "2022-04-30",
 				shortName: "epub-tests",
-				edDraftURI: "https://w3c.github.io/epub-tests/results/",
+				edDraftURI: "https://w3c.github.io/epub-tests/results/drafts/",
                 copyrightStart: "2021",
 				editors:[
 					{
```
Comment (dlazin) [inline docs/drafts/index.html:19]: Unrelated to this PR, but reminder that we were thinking of removing http://www.aptest.com/standards/htmldiff/htmldiff.pl (line 38) from all respec configs.

File: src/lib/html.ts [hunk 2]
```diff
@@ -101,7 +101,12 @@ function create_impl_reports(data: ReportData): string {
             add_child(header_row, 'th', 'Id');
             add_child(header_row, 'th', 'Req');
             for (const impl of (consolidated ? data.consolidated_implementers : data.implementers)) {
-                const head = impl.variant !== 'consolidated' ? `${impl.name}<br />${impl.variant}` : impl.name;
+                let head: string;
+                if (impl.variant !== undefined) {
+                    head = impl.variant !== 'consolidated' ? `${impl.name}<br />${impl.variant}` : impl.name;
```
Comment (dlazin) [inline src/lib/html.ts:106]: I'm having trouble parsing this (especially telling how it's different from the old version) — could use a comment.

**Review by dlazin — APPROVED** [review]
Sorry for the slow review, and thanks for doing this!

File: src/lib/html.ts (hunk 2 above)
Comment (iherman) [reply src/lib/html.ts:106]: The variant field in the implementation report is optional, and the original code did not take that into account. As a result, in some cases, an extra "(undefined)" field appeared in the output. Bug...

File: docs/drafts/index.html (hunk 1 above)
Comment (iherman) [reply docs/drafts/index.html:19]: You are right. I have removed these before merging.

**Pull request merged**